PensionManagementUserLoginService/Models/Context/AuthDbContext.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+
+namespace PensionManagementUserLoginService.Models.Context
+{
+    public class AuthDbContext: IdentityDbContext
+    {
+        public AuthDbContext(DbContextOptions<AuthDbContext> options) : base(options)
+        {
+
+        }
+        protected override void OnModelCreating(ModelBuilder builder)
+        {
+            base.OnModelCreating(builder);
+            var readerRoleId = "92327fd8-20fc-42d8-9bd8-536a746bc3ec";
```

Comment:
is this the default id or you have provide it

Reply:
we have generated new Guid Id 
where it is a unique identifier for the readerroleId

Reply:
Guid.New().ToString('N');